README.md

```diff
@@ -1389,6 +1478,20 @@ $secureConnector = new React\Socket\SecureConnector($dnsConnector, null, array(
 ));
 ```
 
+Opportunistic TLS is supported by the secure connector by using `opportunistic-tls://` as scheme instead of `tls://`. This, when 
+connected, returns a [`OpportunisticTlsConnectionInterface`](#opportunistictlsconnectioninterface) instead of the default 
+[`ConnectionInterface`](#connectioninterface) and won't be TLS encrypted from the start so you can enable the TLS 
+encryption on the connection after negotiating with the server.
```

Comment:
```suggestion
The `SecureConnector` class supports opportunistic TLS  by using `opportunistic-tls://` as scheme instead of `tls://`. This, when connected, returns a [`OpportunisticTlsConnectionInterface`](#opportunistictlsconnectioninterface) instead of the default [`ConnectionInterface`](#connectioninterface). It won't be TLS encrypted from the start, but you can enable the TLS encryption on the connection after negotiating with the server.
```